stl/inc/queue

```diff
@@ -311,6 +360,19 @@ template <class _Pr, class _Container, class _Alloc,
                     uses_allocator<_Container, _Alloc>>,
         int> = 0>
 priority_queue(_Pr, _Container, _Alloc) -> priority_queue<typename _Container::value_type, _Container, _Pr>;
+
+template <class _Iter, class _Alloc, class _Container = vector<_Iter_value_t<_Iter>, _Alloc>,
+    enable_if_t<conjunction_v<_Is_iterator<_Iter>, _Is_allocator<_Alloc>, uses_allocator<_Container, _Alloc>>, int> = 0>

```

Comment:
I am wondering whether we actually need the `_Is_allocator<_Alloc>` here or if it is subsumed by the `uses_allocator` trait

Reply:
Adopted P1518R2